File: Employees-Management-System/src/app/Models/employeePost.model.ts
export class EmployeePostModel {
  firstName: string;
  lastName: string;
  identity: string;
  gender: Gender = Gender.Male; // ערך דיפולטיבי
  birthdate: string;
  employmentStartDate: string;

  constructor(
    firstName: string,
    lastName: string,
    identity: string,
    birthdate: string,
    employmentStartDate: string,
    gender: Gender = Gender.Male // ערך דיפולטיבי
  ) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.identity = identity;
    this.birthdate = birthdate;
    this.employmentStartDate = employmentStartDate;
    this.gender = gender;
  }
}

  
  export enum Gender {
    Male = 0,
    Female = 1
  }
  

File: Employee Management/EmployeeManagement/EmployeeManagement.API/Models/EmployeePostModel.cs
using EmployeeManagement.Core.Models;

namespace EmployeeManagement.API.Models
{
    public class EmployeePostModel
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Identity { get; set; }
        public Gender Gender { get; set; }
        public DateTime Birthdate { get; set; }
        public DateTime EmploymentStartDate { get; set; }
    }
}
